python/docs/source/user_guide/pandas_on_spark/faq.rst

```diff
@@ -2,30 +2,25 @@
 FAQ
 ===
 
-Is it Koalas or koalas?
------------------------
+Should I use PySpark's DataFrame API or pandas API on Spark?
+------------------------------------------------------------
 
-It's Koalas. Unlike pandas, we use upper case here.
+If you are already familiar with pandas and want to leverage Spark for big data, we recommend using pandas API on Spark.
+If you are learning Spark from ground up, we recommend you start with PySpark's API.
 
-Should I use PySpark's DataFrame API or Koalas?
------------------------------------------------
-
-If you are already familiar with pandas and want to leverage Spark for big data, we recommend
-using Koalas. If you are learning Spark from ground up, we recommend you start with PySpark's API.
-
-Does Koalas support Structured Streaming?
+Does pandas API on Spark support Structured Streaming?
 -----------------------------------------
```

Comment:
@itholic can you fix this underbars? it causes warnings:

```
/.../spark/python/docs/source/user_guide/pandas_on_spark/faq.rst:12: WARNING: Title underline too short.

Does pandas API on Spark support Structured Streaming?
-----------------------------------------
/.../spark/python/docs/source/user_guide/pandas_on_spark/faq.rst:12: WARNING: Title underline too short.

Does pandas API on Spark support Structured Streaming?
-----------------------------------------
```

Reply:
I don't know how the CI passed since we turn warnings as errors.

Reply:
Yes, I'll fix it in the follow-up